pages/02.Tutorials/10.cinder-multiattach/docs.en.md

```diff
@@ -0,0 +1,161 @@
+| a6d64bf4-d2da-4d17-ada0-9b5c10942bc9 | App Instance 3 | ACTIVE | net_demo=192.168.1.6                 | Ubuntu Xenial 16.04 (2020-03-30) |
++--------------------------------------+----------------+--------+----------------------------------------+----------------------------------+
+
+openstack --os-compute-api-version 2.60 server add volume "App Instance 1" testvolume
+openstack --os-compute-api-version 2.60 server add volume "App Instance 2" testvolume
+openstack --os-compute-api-version 2.60 server add volume "App Instance 3" testvolume
+
+```
+
+If you get an error message like
+>versions supported by client: 2.1 - 2.41
+
+you need to update your openstack client. Working with multi attach volumes is supported from nova API version 2.60.
+
+
+### Setup OCFS2
+
+Multi-attach volumes require a [cluster filesystem](https://en.wikipedia.org/wiki/Clustered_file_system#SHARED-DISK) like OCFS2 or GFS2 to coordinate concurrent filesystem access.
+
+**Warning: We do not recommend to use a file system like Ext4 or XFS for multi-attach volumes. Concurrent write access will destroy the filesystem and the data on that volume.**
+
+For this tutorial we use OCFS2.
+To setup OCFS2 we need to install the ocfs2-tools and kernel module for the running kernel version on every VM:
+
+```shell
+apt install ocfs2-tools linux-modules-extra-4.15.0-91-generic
+```
+
+OCFS2 is configurend in /etc/default/o2cb and /etc/ocfs/cluster.conf.
+Set `O2CB_ENABLED=true` in /etc/default/o2cb. All other settings can left unchanged.
+
+```shell
+# O2CB_ENABLED: 'true' means to load the driver on boot.
+O2CB_ENABLED=true
+
+[ ... ]
+```
+
+Define all 3 VMs in /etc/ocfs2/cluster.conf to tell o2cb which nodes are part of the cluster and how to reach them. Make sure that the `name` setting for every node matches the hostname and that the `number` is unique in the cluster.
+This file is identical on all 3 VMs:
+
+```shell
+# /etc/ocfs2/cluster.conf
+node:
+        ip_port = 7777
+        ip_address = 192.168.1.14
+        number = 1
+        name = app-instance-1
+        cluster = ocfs2
+node:
+        ip_port = 7777
+        ip_address = 192.168.1.2
+        number = 2
+        name = app-instance-2
+        cluster = ocfs2
+node:
+        ip_port = 7777
+        ip_address = 192.168.1.6
+        number = 3
+        name = app-instance-3
+        cluster = ocfs2
+
+cluster:
+    node_count = 3
+    name = ocfs2
+
+```
+
+Start the o2cb service on all 3 VMs and check the status:
+
+```shell
+systemctl start o2cb
+systemctl status o2cb
+systemctl enable o2cb
+```
+
+### Create a filesystem on the volume
+
+In this example there is only one cinder volume attached to the VMs. The device name on all VMs is `/dev/vdb`.
+The next step os only necessary on one VM:
+
+```shell
+mkfs.ocfs2 /dev/vdb
+```
+
+### Mount the filesystem
+
+Mount the filesystem on all 3 VMs:
+
+```shell
+mount.ocfs2 /dev/vdb /mnt
+```
+
+Now it is possible to read and write files from all VMs to the volume
```

Comment:
Missing period at the end. And "...to the volume" doesn't sound right. Maybe "Now it is possible to read and write files on the volume from all VMs."

Reply:
yes, that sounds better